pkg/cli/repositories.go

```diff
@@ -0,0 +1,447 @@
+package cli
+
+import (
+	"bytes"
+	"crypto/md5"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/vorteil/vorteil/pkg/vpkg"
+)
+
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2020 vorteil.io Pty Ltd
+ **/ //
+
+var repositoriesCmd = &cobra.Command{
+	Use:   "repositories",
+	Short: "Interact with vorteil repositories",
+}
+
+var keysCmd = &cobra.Command{
+	Use:   "keys",
+	Short: "Create, List and Delete keys for authentication with Vorteil Repositories",
+}
+
+var defaultKeyCmd = &cobra.Command{
+	Use:   "default",
+	Short: "Prints what the current default is",
+	Run: func(cmd *cobra.Command, args []string) {
+		usr, err := os.UserHomeDir()
+		if err != nil {
+			SetError(err, 1)
+			return
+		}
+
+		pathCheck := filepath.Join(usr, ".vorteil", "repository-keys")
+
+		if _, err = os.Stat(pathCheck); os.IsNotExist(err) {
+			fmt.Printf("no keys found\n")
```

Comment:
use the cli logger to print and maybe set error code. Also `no keys found` should print if **repository-keys** or if **repository-keys** is empty.

Reply:
I also noticed this same code block is in the list command, It might be worth just making a util function and using that in both functions.